lib/cosmos/io/json_drb.rb

```diff
@@ -90,7 +96,25 @@ def start_service(hostname = nil, port = nil, object = nil)
         @thread = Thread.new do
           begin
             while true
-              socket = @listen_socket.accept()
+              begin
+                socket = @listen_socket.accept_nonblock
+              rescue Errno::EAGAIN, Errno::ECONNABORTED, Errno::EINTR, Errno::EWOULDBLOCK
```

Comment:
I remember seeing some new code to handle Mac specific exceptions recently. Is this error list complete?